src/hotspot/share/classfile/vmSymbols.hpp

```diff
@@ -137,6 +140,7 @@ class SerializeClosure;
   template(java_util_Properties,                      "java/util/Properties")                     \
   template(java_util_DualPivotQuicksort,              "java/util/DualPivotQuicksort")             \
   template(jdk_internal_misc_Signal,                  "jdk/internal/misc/Signal")                 \
+  template(jdk_internal_math_Float16Math,             "jdk/internal/math/Float16Math")            \
```

Comment:
This seems to be leftover template.

Reply:
I don't see use of this one, you have another one with jdk_internal_vm_vector_Float16Math which is being used.